Maestro/subscriptions.json

```diff
@@ -634,8 +634,7 @@
       "triggerPaths": [
         "https://github.com/aspnet/AspLabs/blob/master/**/*",
         "https://github.com/aspnet/AspLabs/blob/release/**/*",
-        "https://github.com/dotnet/aspnetcore/blob/blazor-wasm/**/*",
-        "https://github.com/dotnet/aspnetcore/blob/master/**/*",
+        "https://github.com/dotnet/aspnetcore/blob/main/**/*",
```

Comment:
No longer have a blazor-wasm branch and will not have a main branch a bit after 5PM